editor/editor_settings.cpp

```diff
@@ -600,7 +600,7 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 
 	// Behavior: Files
 	_initial_set("text_editor/behavior/files/trim_trailing_whitespace_on_save", false);
-	_initial_set("text_editor/behavior/files/autosave_interval_secs", 0);
+	EDITOR_SETTING(Variant::INT, PROPERTY_HINT_RANGE, "text_editor/behavior/files/autosave_interval_secs", 0, "0,600,1,or_greater");
```

Comment:
You can tell this setting was here way before suffixes in 4.0.
```suggestion
	EDITOR_SETTING(Variant::INT, PROPERTY_HINT_RANGE, "text_editor/behavior/files/autosave_interval_secs", 0, "0,600,1,or_greater,suffix:s");
```